src/users/infra/repositories/transformers/apiTokenInfoTransformers.ts

```diff
@@ -0,0 +1,26 @@
+import { AxiosResponse } from 'axios'
+import { ApiTokenInfo } from '../../../domain/models/ApiTokenInfo'
+
+export const transformGetApiTokenResponseToApiTokenInfo = (
+  response: AxiosResponse
+): ApiTokenInfo => {
+  const apiTokenInfoPayload = response.data.data
+  const messageParts = apiTokenInfoPayload.message.split(' ')
+  const expirationDateFormattedTimestamp = `${messageParts[4]}T${messageParts[5]}`
```

Comment:
This is ok but in the future we could modify the API to return a JSON object with two properties - `token` and `expirationDate` instead of just a string which makes it more difficult to obtain the response data.

Reply:
I totally agree, and mentioned what you suggest on the API extension PR: https://github.com/IQSS/dataverse/pull/10858

I've kept the string messages as responses for backward compatibility.